src/ledger/LedgerManagerImpl.cpp

```diff
@@ -308,7 +314,8 @@ LedgerManagerImpl::loadLastKnownLedger(
         }
         else
         {
-            advanceLedgerPointers();
+            LedgerState ls(mApp.getLedgerStateRoot());
```

Comment:
`ls` shadows previous declaration (probably better to refactor this function anyways as to not have the previous one in scope at all)